quickstart/environment.yml

```diff
@@ -0,0 +1,8 @@
+name: coiled
```

Comment:
@FabioRosado is this file exercised anywhere, or referenced in the docs? Currently the software environments are specified in `**/create_notebook.py`. I was planning to move that out into `environment.yml` files, but have not yet.

Reply:
Thank you for the question, this was a leftover from the chat with Kris - we aren't using that file anywhere in the notebook so feel free to remove or reuse it as you please 😄 👍